internal/config/registry.go

```diff
@@ -22,7 +24,14 @@ type registry struct {
 	Auth                  []RegistryCredentials `yaml:"auth" json:"auth" mapstructure:"auth"`
 }
 
-func (cfg *registry) parseConfigValues() {
+func (cfg registry) loadDefaultValues(v *viper.Viper) {
+	v.SetDefault("registry.insecure-skip-tls-verify", false)
+	v.SetDefault("registry.insecure-use-http", false)
+	v.SetDefault("registry.auth", []RegistryCredentials{})
+}
+
+// nolint: unparam
+func (cfg *registry) parseConfigValues() error {
```

Comment:
question: I looked up `unparam` on the [golangci-lint docs](https://golangci-lint.run/usage/linters/) — could it be we no longer need this? The descrption of `unparam` is:

>Reports unused function parameters

Reply:
actually it is still required such that this fulfills the `parser` interface (unparam applies to return values as well, in this case is always nil)